apps/docs/.vitepress/config.ts

```diff
@@ -421,6 +421,10 @@ export default defineConfig({
         link: '/guide/tooling/',
         collapsed: false,
         items: [
+          {
+            text: 'GraphQL Integration',
```

Comment:
I wanted to put this at the end of the array because it's arguably less important than testing and the CLI, but there is a [bug](https://github.com/FuelLabs/github-actions/issues/30) that prevented me from doing it.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1982

Added a PR for a fix that allowed me to mitigate this one (I hope - works locally).

Reply:
Still [fails](https://github.com/FuelLabs/fuels-ts/actions/runs/8505865042/job/23295073113?pr=1982#step:6:1177) with the same problem.